renderdoc/api/replay/renderdoc_replay.h

```diff
@@ -520,13 +522,16 @@ DOCUMENT(R"(Create a :class:`WindowingData` for an Wayland ``wl_surface`` handle
 :return: A :class:`WindowingData` corresponding to the given window.
 :rtype: WindowingData
 )");
-inline const WindowingData CreateWaylandWindowingData(wl_display *display, wl_surface *window)
+inline const WindowingData CreateWaylandWindowingData(wl_display *display, wl_surface *window,
+                            uint32_t *width, uint32_t *height)
```

Comment:
Windowing data must not include dimensions and definitely not arbitrary pointers to user supplied values. These should instead be queried from the window handle itself.